app/models/insights_missing_hosts.rb

```diff
@@ -0,0 +1,4 @@
+class InsightsMissingHosts < ApplicationRecord
```

Comment:
```suggestion
class InsightsMissingHost < ApplicationRecord
```
Models should be singular.